updater/lib/dependabot/updater/error_handler.rb

```
@@ -90,9 +90,9 @@ def handle_job_error(error:, dependency_group: nil)
 
       # Provides logging for errors that occur outside of a dependency context
       def log_job_error(error:, error_type:, error_detail: nil)
-        if error_type == "unknown_error"
+        if error_type == "unknown_error" && !Dependabot.enterprise?
```

Instead of checking if we are on `enterprise?` here, we can just always post to the `unknown_error` endpoint and check if we are on enterprise on the backend. If we are, then we can return 200 and discard the error message since we can't use that info.

---

```suggestion
        if error_type == "unknown_error"
```